src/converter.py

```diff
@@ -0,0 +1,173 @@
+import datetime
+import json
+
+def convert_tes_to_wrroc(tes_data):
+    try:
```

Comment:
You should wrap only the code that you expect to raise particular errors in `try`/`except` - not just a bunch of commands and catch some generic errors. There is no added benefit of catching, say, an indexing error and then just say that there was an indexing error - the regular error and traceback will have all of that info already - and more.